{"name": "相乘", "status": "failed", "statusDetails": {"message": "AssertionError: assert (1 * 1) == 2", "trace": "self = <test_case.test_func1.TestLogin object at 0x00000239490D6F70>, a = 2\n\n    @allure.title('相乘')\n    @pytest.mark.parametrize('a', [1, 2, 3])\n    def test_pare(self, a):\n        print('----test_pare----')\n>       assert 1 * 1 == a  # 断言\nE       assert (1 * 1) == 2\n\ntest_func1.py:26: AssertionError"}, "parameters": [{"name": "a", "value": "2"}], "start": 1611325858839, "stop": 1611325858839, "uuid": "349f0e32-de7d-4c75-960a-a9e1920636c7", "historyId": "c9930f9207d6bf4aa55bfc3b88742e56", "testCaseId": "2a999addb8e1ee6be8ffc6526daab9e1", "fullName": "test_case.test_func1.TestLogin#test_pare", "labels": [{"name": "feature", "value": "登录功能"}, {"name": "parentSuite", "value": "test_case"}, {"name": "suite", "value": "test_func1"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "steven"}, {"name": "thread", "value": "8912-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.test_func1"}]}